### ISSUE TITLE
"Tech Talks" are incorrectly appended with a year in the TableView

### ISSUE THREAD

**Issue by bcmn** [insidegui/WWDC#504]
Sessions belonging to the "Tech Talks", "Developer Insights" & "App Store Connect" events get incorrectly appended with a year in the TableView on the left, as shown below:

![image](https://user-images.githubusercontent.com/11646957/45539325-4a8c8000-b801-11e8-9a37-345546dd9c55.png)

Specifically, the issue is related to [this function](https://github.com/insidegui/WWDC/blob/master/WWDC/SessionViewModel.swift#L192) in `SessionViewModel.swift`:
```
    static func subtitle(from session: Session, at event: ConfCore.Event?) -> String {
        guard let event = event else { return "" }

        let year = Calendar.current.component(.year, from: event.startDate)

        // Currently, there's only one event which is not a WWDC (the "Fall 2017" event),
        // for some reason it includes the year on its name, while WWDC editions do not,
        // so we have to use this workaround to avoid displaying the year twice
        let name = event.name.replacingOccurrences(of: " \(year)", with: "")

        return "\(name) \(year) · Session \(session.number)"
    }
```

We only want to append the year to "WWDC" event names, without altering any other event names.

**Comment by marcocapano** [comment]
I'm on it :) 😊

**Comment by marcocapano** [comment]
PR submitted🎉

**Comment by insidegui** [comment]
Thanks @marcocapano

**Comment by marcocapano** [comment]
Since a PR is already on this I guess we could close it?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by bcmn** [comment]
Typically, we close issues once the PR is merged -- especially as there's a reference to the PR above. It's possible to use [Github's keywords](https://help.github.com/articles/closing-issues-using-keywords/) to make this even easier, & even do it automatically.

**Comment by insidegui** [comment]
Implemented in #506
